themes/default/theme.lua

```diff
@@ -2,6 +2,11 @@
 -- Default awesome theme --
 ---------------------------
 
+local beautiful = require("beautiful")
+local xresources = beautiful.xresources
+local dpi = xresources.apply_dpi
+local theme_assets = beautiful.theme_assets
```

Comment:
Isn't this another cyclic dependency waiting to become a problem? It's no problem right now, but can become one in the future (beautiful depends on theme which depends on beautiful).

I'd propose not to `require("beautiful")` here, but directly the needed parts.